core/storage/suggestion/gae_models.py

```diff
@@ -684,29 +684,35 @@ def get_reviewable_translation_suggestions_for_single_exploration(
 
         Returns:
             Tuple of (results, next_offset). Where:
-                results: list(SuggestionModel). A list of all suggestions that
-                    are in-review, not authored by the supplied user, matching
-                    the supplied language code, and correspond to the
-                    given exploration ID.
+                results: list(SuggestionModel). A list of all suggestions
+                    ordered by descending creation date that are in-review,
```

Comment:
This is worded quite confusingly. Use the previous wording and add a sentence at the end saying "The suggestions are ordered by ...". Sometimes splitting up into multiple sentences is clearer.

Reply:
Done, thanks.